src/components/SongPage/ResetAllVotes/ResetAllVotes.js

```diff
@@ -0,0 +1,21 @@
+import React from "react"
+import Button from 'components/common/Button'
+import showIfAuthenticated from "../../../util/showIfAuthenticated";
+
+
+const styles = {
+  container:{
+    display:'flex',
+    justifyContent:'center'
+  }
+}
+const ResetAllVotes = ({resetAllVotes}) => {
+  return (
+  <div style={styles.container}>
+    <Button onClick={ resetAllVotes }>
+      Reset All Votes
+    </Button>
+  </div>
+)}
+
+export default showIfAuthenticated(ResetAllVotes)
```

Comment:
It might make sense to keep `showIfAuthenticated` with the other HOCs in `src/components/SongPage/ResetAllVotes/index.js`.

Reply:
Good call! Change made